bin/cyhy-ip

```diff
@@ -110,7 +111,22 @@ def get_special_intersections(cidrs):
             results[description] = intersection
     return results
 
-
+def has_restricted_ips(nets):
+    reader = geoip2.database.Reader('/usr/local/share/GeoIP/GeoIP2-City.mmdb')
+    bad_countries = ['China','Russia','Iran','North Korea']
+    has_bad_address = False
+
+    for cidr in nets.iter_cidrs():
+        geo_data = reader.city(str(cidr[0]))
+        country_name = geo_data.country.name
+        if country_name in bad_countries:
+            print('{} contains addresses in country that is unavailable due to current scanning policy: {}'.format(cidr,country_name))
+            has_bad_address = True
+    if has_bad_address:
+        return True
+    else:
+        return False
+  
```

Comment:
I don't like the same code being repeated in two places.  Is there any way to put this code in a common place?

Reply:
Yeah, I didn't like copying the code over twice but I am trying to think of how to do it so both files can access it.

Reply:
We'll see if @cisagov/team-ois (the other devs) have any ideas as well.  I'm not very familiar with this part of the CyHy code, so I don't know the best place to move it to.

Reply:
Is there any reasons not to add a single CIDR argument method to `GeoLocDB` in https://github.com/cisagov/cyhy-core/blob/develop/cyhy/core/geoloc.py ? That class is already in use by both tools, and adding it there would make it available for any future needs as part of the library.

Reply:
I am on board with @mcdonnnj's suggestion of adding a method to [`GeoLocDB`](https://github.com/cisagov/cyhy-core/blob/develop/cyhy/core/geoloc.py) to handle this lookup- something like`check_restricted_cidr()`. 👍

Reply:
Yes, that seems like a good solution

Reply:
Added a commit to my branch with the requested changes made. Does this look good?